app/src/main/java/org/oppia/app/topic/train/TopicTrainFragment.kt

```diff
@@ -21,4 +21,9 @@ class TopicTrainFragment : InjectableFragment() {
   override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
     return topicTrainFragmentPresenter.handleCreateView(inflater, container)
   }
+
+  fun submitButtonClicked(){
```

Comment:
This appears incomplete. Please finish.

Reply:
Note that this would conceivably intent into another activity. We should at least introduce that activity as a prerequisite for this issue so that it can be intented into, even if it's temporary.

UI structure work should include ensuring all components are properly connected together (controllers & UIs, and between UI components for in-app navigation).

Reply:
Introduced `QuestionPlayerActivity` which contains `QuestionPlayerFragment` and using intent transferred data to `QuestionPlayerActivity`. Also, linked this to #159 in TODO comment.

Reply:
Finished this functionality and introduced `QuestionPlayerActivity`. Also, Test case for this new activity i in #204